### PR TITLE
Add Tem's research projects

### PR DESCRIPTION
**Pull request by Kundjanasith**
- /project/improve-dc-lstm --> HPCMASPA'2019
- /project/model-compression-retraining --> IJCNN'2020
- /project/federated-heterogeneous --> ICMLA'2020

### CODE REVIEW

**Comment by keichi** [comment]
Maybe we should add a new tag like "Machine Learning"?

File: content/en/project/federated_heterogeneous/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Ensembling Heterogeneous Models for Federated Learning"
+summary: ""
+authors: [Kundjanasith Thonglek]
+tags: []
```
Comment (KoheiIchikawa) [inline content/en/project/federated_heterogeneous/index.md:7]: Add some tags.
The tag will be appeared as an keyword to filter the research topics.
How about just add:
tags: ["Machine Learning"]

File: content/en/project/improve-dc-lstm/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Improving Resource Utilization of Data Center using LSTM"
+summary: ""
+authors: [Kundjanasith Thonglek]
+tags: []
```
Comment (KoheiIchikawa) [inline content/en/project/improve-dc-lstm/index.md:7]: Add some tags.
tags: ["Machine Learning"]

File: content/en/project/model-compression-retraining/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+---
+# Documentation: https://sourcethemes.com/academic/docs/managing-content/
+
+title: "Retraining Quantized Neural Networks without Labeled Data"
+summary: ""
+authors: [Kundjanasith Thonglek]
+tags: []
```
Comment (KoheiIchikawa) [inline content/en/project/model-compression-retraining/index.md:7]: Add some tags.
tags: ["Machine Learning"]

**Review by KoheiIchikawa — COMMENTED** [review]
Yes, we should add a ne tag, "Machine Learning".

You can use some figures as featured images.
Change the figure's name to featured.jpg/png.

**Comment by KoheiIchikawa** [comment]
You should also add links of the papers for each publication using DOI.

**Comment by Kundjanasith** [comment]
@KoheiIchikawa @keichi I have already updated the articles.
I can not add the DOI of the ICMLA'2020 paper because it has not been published yet.

**Comment by KoheiIchikawa** [comment]
## Add symlinks from ja content
Could you add symlinks for your article from ja folder?
```
cd content/ja/project/
ln -s ../../en/project/title-of-your-project1
ln -s ../../en/project/title-of-your-project2
ln -s ../../en/project/title-of-your-project3
```

## Add a filter keyword, "Machine Learning", in projects.md (for both en and ja)
Edit
- content/ja/research/projects.md
- content/en/research/projects.md

And, add the following lines between "Design Pattern" and "Programming Education".
```
  [[content.filter_button]]
    name = "Machine Learning"
    tag = "Machine Learning"
```

**Comment by Kundjanasith** [comment]
I added the symlinks and a filter keyword.

**Review by KoheiIchikawa — APPROVED** [review]

**Pull request merged**